app/views/planning_application/assessment_details/edit.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :page_title do %>
+  <%= t("assessment_details.edit_#{@assessment_detail.category}") %> - <%= t('page_title') %>
+<% end %>
+
+<% add_parent_breadcrumb_link "Home", planning_applications_path %>
+<% add_parent_breadcrumb_link "Application", planning_application_path(@planning_application) %>
+<% add_parent_breadcrumb_link "Assess application", planning_application_assessment_tasks_path(@planning_application) %>
```

Comment:
There's a [shared partial for this](https://github.com/unboxed/bops/blob/main/app/views/shared/_assessment_task_breadcrumbs.html.erb) I made earlier!